tutorial/blending-tutorial/preparing-the-views/README.md

```diff
@@ -0,0 +1,316 @@
+# Declarative Shadow DOM
+
+When blending, we combine view-models and views from different apps. Combining view-models is easy:
```

Comment:
By "we" you mean "we at Starcounter". I believe that the fragment about combining view-models is an implementation detail. The person doing this tutorial really doesn't need to know it unless they ask "how the heck does blending work?". But that should be answered in the Guides, not in the Tutorial.